graders/python/python_autograder/pl_helpers.py

```diff
@@ -0,0 +1,88 @@
+import io
+import urllib
+import base64
+from functools import wraps
+from code_feedback import Feedback
+import pygments
+from pygments.lexers import PythonLexer
+from pygments.formatters import Terminal256Formatter
+
+
+class DoNotRun(Exception):
```

Comment:
@nicknytko I can't find this being used anywhere. Was this a pre-cursor to `not_allowed`?

Reply:
This is used by the `not_repeated` test decorator ... which I should probably document and add instructions on how to set up test cases that are run for multiple iterations.